packages/components/src/Filter/FilterDatePicker/_docs/FilterDatePicker.stories.tsx

```diff
@@ -3,12 +3,12 @@ import { Meta, StoryObj } from "@storybook/react"
 import Highlight from "react-highlight"
 import { Paragraph } from "@kaizen/typography"
 import { defaultMonthControls } from "~components/Calendar/_docs/controls/defaultMonthControls"
-import { renderTriggerControls } from "~components/Filter/_docs/controls/renderTriggerControls"
+import { renderTriggerControls } from "~components/Filter/Filter/_docs/controls/renderTriggerControls"
```

Comment:
Oh actually these shared controls can now move to the top level :) It was actually just shared stuff for the `Filter*` components, and not for the `Filter` component itself.